docs/ai/conceptual/embeddings.md

```diff
@@ -0,0 +1,75 @@
+
+Summarizing text is especially useful for [prompt engineering](/), because it gives you a way to use far fewer tokens to repesent a given text. You use instructions in prompt engineering to tell an LLM what to do, and can include text content with the instruction to give the model more context. However, LLMs limit the number of tokens per input, which constrains how much text you can include in prompts. If you try to include more than the limit in one input, the model will ignore some or all of that input. Some LLMs [charge per token](/pricing/details/cognitive-services/openai-service/) or have [quota systems](/azure/ai-services/openai/quotas-limits#regional-quota-limits), more reasons to watch your token counts. If you create an embedding for the text content you want to include, the LLM can find a shorter piece of text that has a similar vector, so you can use it instead of the original text to reduce the number of tokens you use to represent the content's meaning.
+
+## Embeddings preserve semantic meaning
+
+Embeddings turn non-numeric data into numeric data that represents semantic meaning. Specifically, an embedding is a vector&mdash;an array of numeric values&mdash; that an LLM uses to compare semantic meanings. Two embeddings with similar vectors also have similar semantic meanings, such as an image and its alt text, or a course syllabus and the textbook for that course.
+
+### Choose an embedding model
+
+You generate embeddings for your raw data by using an embedding model: an AI model that can encode a piece of non-numeric data into a vector (a long array of numbers). The model can also decode an embedding into non-numeric data that has the same or similar meaning as the original, raw data.
+
+In Semantic Kernel, you use connectors to integrate embedding models and databases into your kernel, so you can quickly pull in text data, and generate and store embeddings. This lets you use a vector database solution to store and retrieve semantic memories.
+
+In Azure OpenAI, the service includes access to [embedding models](/azure/ai-services/openai/concepts/models#embeddings).
+
+### Store and process embeddings in a vector database
+
+Vector databases are designed to store and process vectors, so they're a natural home for embeddings. Different vector databases offer different processing capabilities, so you should choose one based on your raw data and your goals. For information about your options, see [available vector database solutions](vector-databases.md#available-vector-database-solutions).
+
+## Use cases for embeddings
+
+This section lists the main use cases for embeddings. LLMs can process text embeddings, but to use other data types you need to include a suitable embedding model, such as **dall-e-3** for image generation.
```

Comment:
I think dall-e-3 is a generation model, not an embedding model. I think dall-e-3 uses CLIP as it's encoder. I found this research paper that talks about it: https://cdn.openai.com/papers/dall-e-3.pdf

Reply:
Correction: Dall-E-3 uses "CLIP ViT-B/32" a very specific model for mapping text and images into the same vector space. If you want to talk about a more straight-forward image embedding model I'd suggest ViT: https://huggingface.co/docs/transformers/main/en/model_doc/vit

Reply:
Thank you! I wanted to link to the embedding model itself but I didn't want to link to CLIP given that Dall-E uses a custom version. ViT it is!